src/main/resources/site/content-types/highchart/highchart.ts

```diff
@@ -20,7 +20,7 @@ export interface Highchart {
   graphType: "line" | "pie" | "column" | "bar" | "area" | "barNegative";
 
   /**
-   * Antall desimalplasser som vises
+   * Vis desimaler for avlesingspunktene i diagrammet, antall:
```

Comment:
We should try to avoid norwegian comments in the sourcecode. Probably something to discuss in one of our meeting.

Reply:
Agree! Also for generated files.